helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -335,11 +336,17 @@ internal fun OrtResult.getUnresolvedRuleViolations(): List<RuleViolation> {
     }
 }
 
+/**
+ * Return a copy with the [LicenseFindingCuration]s replaced by the given scope excludes.
+ */
+internal fun RepositoryConfiguration.replaceLicenseFindingCurations(curations: List<LicenseFindingCuration>):
+    RepositoryConfiguration = copy(curations = (this.curations ?: Curations()).copy(licenseFindings = curations))
+
 /**
  * Return a copy with the [PathExclude]s replaced by the given scope excludes.
  */
-internal fun RepositoryConfiguration.replacePathExcludes(pathExcludes: List<PathExclude>): RepositoryConfiguration =
-    copy(excludes = (excludes ?: Excludes()).copy(paths = pathExcludes))
+internal fun RepositoryConfiguration.replacePathExcludes(pathExcludes: List<PathExclude>):
+    RepositoryConfiguration = copy(excludes = (excludes ?: Excludes()).copy(paths = pathExcludes))
```

Comment:
Same as above, either keep the return type on the same line as the rest of the signature or use the mulit-line syntax.

Reply:
Done.